pkg/synthetictests/allowedalerts/types.go

```diff
@@ -1,41 +1,12 @@
 package allowedalerts
 
 import (
-	"bytes"
 	_ "embed"
 	"sync"
 
 	"github.com/openshift/origin/pkg/synthetictests/historicaldata"
 )
 
-const (
-	// p95Query produces the query_results.json.  Take this query and run it against bigquery, then export the results
-	// as json and place them query_results.json.
-	// This query produces the p95 and p99 alert firing seconds for the named alerts on a per platform, release, topology,
-	// network type basis.
-	p95Query = `
-SELECT * FROM openshift-ci-data-analysis.ci_data.Alerts_Unified_LastWeek_P95
-where
-  alertName = "etcdMembersDown" or 
-  alertName = "etcdGRPCRequestsSlow" or 
-  alertName = "etcdHighNumberOfFailedGRPCRequests" or 
-  alertName = "etcdMemberCommunicationSlow" or 
-  alertName = "etcdNoLeader" or 
-  alertName = "etcdHighFsyncDurations" or 
-  alertName = "etcdHighCommitDurations" or 
-  alertName = "etcdInsufficientMembers" or 
-  alertName = "etcdHighNumberOfLeaderChanges" or 
-  alertName = "KubeAPIErrorBudgetBurn" or 
-  alertName = "KubeClientErrors" or 
-  alertName = "KubePersistentVolumeErrors" or 
-  alertName = "MCDDrainError" or 
-  alertName = "PrometheusOperatorWatchErrors" or
-  alertName = "VSphereOpenshiftNodeHealthFail"
-order by 
- AlertName, Release, FromRelease, Topology, Platform, Network
-`
-)
```

Comment:
This query was removed, it's a duplicate copy of the one in ci-tools, and has surely drifted since being recorded here.

Reply:
I like the idea of removing this esp. if it's outdated.  I'm assuming you are referring to the query in [ci_data_client.go](https://github.com/openshift/ci-tools/blob/82c90e69750acdd9b37b766da424bc9e860d0277/pkg/jobrunaggregator/jobrunaggregatorlib/ci_data_client.go#L163-L170).

In place of it, can you please add some comment that mentions that `query_results.json` is created from big query and updated using automated PRs.